saleor/plugins/manager.py

```diff
@@ -797,6 +798,41 @@ def list_payment_gateways(
             )
         return gateways
 
+    def list_shipping_methods(
```

Comment:
IMHO we should have separate methods for `list_shipping_methods_for_order` and `list_shipping_methods_for_checkout`. The same thing for get_shipping_method (if `checkout` there is really important)